src/main/java/org/squiddev/plethora/api/EntityWorldLocation.java

```diff
@@ -29,13 +33,13 @@ public World getWorld() {
 	@Nonnull
 	@Override
 	public BlockPos getPos() {
-		return new BlockPos(entity.posX, entity.posY + entity.getEyeHeight(), entity.posZ);
+		return new BlockPos(entity.posX, entity.posY + (isEntityLookingDown() ? 0 : entity.getEyeHeight()), entity.posZ);
```

Comment:
I assume this is an attempt to fix `.suck()`? I think it's best to leave off this change for now - it's definitely going to cause more issues than solves, as we base a lot of other things off this (block scanner/entity sensor centre, etc...).

Reply:
I'll make a commit now to revert this, can we think of a list of tests I can perform to ensure future implementations don't break other functionality?

I was a little concerned when I wrote this that it may have side-effects, my thinking was "Eh, just look up and the functionality is normal."

Reply:
Resolved with https://github.com/SquidDev-CC/plethora/pull/218/commits/94d944b42f23bae3160ace988ba495d17c06f3ed